packages/flutter_tools/templates/app_shared/web/index.html.tmpl

```diff
@@ -21,7 +21,7 @@
   <meta name="description" content="{{description}}">
 
   <!-- iOS meta tags & icons -->
-  <meta name="apple-mobile-web-app-capable" content="yes">
+  <meta name="mobile-web-app-capable" content="yes">
```

Comment:
This one was the one I was most interested in, so new flutter web apps don't get shouted at by Chrome for this meta-tag.

(Alternatively we might need to remove these, because they're precursors to the "manifest.json" mechanism for PWA/installing on the home screen)